tools/codegen/gen_lazy_tensor.py

```diff
@@ -129,7 +129,8 @@ def run_gen_lazy_tensor(aten_path: str, source_yaml: str, output_dir: str,
                         # per_operator_headers changes whether ATen/Functions.h or individual operator headers are used
                         # it must match how ATen was built
                         per_operator_headers: bool = False,
-                        backend_name: str = default_args.backend_name) -> None:
+                        backend_name: str = default_args.backend_name,
+                        gen_forced_fallback_code: bool = False) -> None:
```

Comment:
mostly just to make sure i'm following along - doesn't this path only get taken at runtime if an environment variable is set? I'm wondering if there's any reason to make it conditional in the codegen instead of just being conditional on the env variable.

Reply:
I agree with you, but Jack was wanting cleaner generated code; I think we could convince him to accept it, but there was one particular issue:

the header `ts_eager_fallback.h` needed to be included if that code is generated, so using this flag in the codegen lets us skip including that header.  If in the future we make the fallback part of the backend-api, we could consider always generating this codepath.

Reply:
ok, sounds reasonable. The delta is super small anyway.